crates/utils/src/utils.rs

```diff
@@ -148,3 +151,27 @@ pub fn get_ip(conn_info: &ConnectionInfo) -> IpAddr {
       .to_string(),
   )
 }
+
+pub fn clean_url_params(mut url: Url) -> Url {
+  let new_query = url
+    .query_pairs()
+    .filter(|q| !CLEAN_URL_PARAMS_REGEX.is_match(&q.0))
+    .map(|q| format!("{}={}", q.0, q.1))
+    .join("&");
+  url.set_query(Some(&new_query));
+  url
+}
```

Comment:
Nice, not bad. You should consider making this a rust crate too, it'd be good practice for doing that. Plus it could be useful for others.

Reply:
I dont think its worth the effort to publish a crate which has less than 10 lines of code. I'd rather focus on activitypub.